src/components/harvestLogs/HarvestLogDrawer.tsx

```diff
@@ -31,31 +31,18 @@ import { getPickers } from "api/pickers";
 import { getSeasons } from "api/seasons";
 import { useState } from "react";
 import { IPicker } from "project-2-types/lib/pickers";
-
-interface IHarvestLogForm {
-  season: {
-    id: string;
-    label: string;
-    name: string;
-    product: { name: string };
-    price: number;
-  };
-  picker: { id: string; label: string; name: string };
-  collectedAmount: number;
-  totalDeduction: number;
-  productName?: string;
-  seasonDeductionsIds: Array<string>;
-  notes?: string;
-  createdAt?: string;
+import ICreateHarvestLogRequest from "project-2-types/lib/harvestLog.ajv";
+import { ajvResolver } from "@hookform/resolvers/ajv";
+
+function formatDate(date: number): string {
```

Comment:
just a note for later to use the intl message to format the dates

Reply:
Ok yeah will implement that in an upcoming ticket! Thanks!